11.0/base_requirements.txt

```diff
@@ -0,0 +1,67 @@
+# Odoo dependencies
+Babel==2.3.4
+decorator==4.0.10
+docutils==0.12
+ebaysdk==2.1.5
+feedparser==5.2.1
+gevent==1.1.2
+greenlet==0.4.10
+html2text==2016.9.19
+Jinja2==2.8
+lxml==3.5.0
+Mako==1.0.4
+MarkupSafe==0.23
+mock==2.0.0
+num2words==0.5.4
+ofxparse==0.16
+passlib==1.6.5
+Pillow==3.4.1
+psutil==4.3.1
+psycopg2==2.7.1
+pydot==1.2.3
+pyldap==2.4.28
+pyparsing==2.1.10
+PyPDF2==1.26.0
+pyserial==3.1.1
+python-dateutil==2.5.3
+pytz==2016.7
+pyusb==1.0.0
+PyYAML==3.12
+qrcode==5.3
+reportlab==3.3.0
+requests==2.11.1
+six==1.10.0
+suds-jurko==0.6
+vatnumber==1.2
+vobject==0.9.3
+Werkzeug==0.11.11
+XlsxWriter==0.9.3
+xlwt==1.3.*
+xlrd==1.0.0
+
+# In v10 setup but not part of official requirements ?
```

Comment:
Well I updated them to their last version.
And disabled
colorama (compatibility for windows terminal ?)
roman

Did not remove :
simplejson is still present in a server-tools module.

Otherwise, checked a few projects and
gdata not used